public/video-ui/src/components/ScheduledLaunch/ScheduledLaunch.js

```diff
@@ -2,6 +2,16 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import DatePicker from '../FormFields/DatePicker';
 import moment from 'moment';
+import Icon from '../Icon';
+import ScheduleRecap from '../ScheduleRecap/ScheduleRecap';
+
+const isFutureDate = date => date && moment(date).isAfter(moment());
+const isSameOrAfter = (dateA, dateB) =>
+  moment(dateA).isSameOrAfter(moment(dateB));
+const isAfter = (dateA, dateB) => moment(dateA).isAfter(moment(dateB));
+
+const impossiblyDistantDate = moment('2499-01-01T00:00:00+00:00').valueOf();
+const laterThanImpossiblyDistantDate = moment('2500-01-01T00:00:00+00:00').valueOf();
```

Comment:
Not sure why we need both `impossiblyDistantDate` and `laterThanImpossiblyDistantDate`. Can't we just check if a date is not before `impossiblyDistantDate` to find out if it is embargoed indefinitely?

Reply:
👍